ci/pipelines/deploy-to-test.yml

```diff
@@ -526,6 +539,45 @@ jobs:
         file: telegraf-ecr-registry-test/tag
       - load_var: nginx_image_tag
         file: nginx-proxy-ecr-registry-test/tag
+      - load_var: git_tag
+        file: toolbox-git-release/.git/ref
+      - task: create-messages
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: alpine
+          params:
+            APP_NAME: toolbox
+            APP_GIT_TAG: ((.:git_tag)
+            TELEGRAF_GIT_TAG: ((.:telegraf_image_tag)
+          outputs:
+            - name: message
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                cat <<EOT >> message/success
+                :green-circle: Deployment of ${APP_NAME} was successful :tada:
```

Comment:
I suppose this could mention the environment as well (same for the error scenario)?

Reply:
yes you're right it should mention the environment 😬